### PR TITLE
Step two

### PR DESCRIPTION
**Pull request by jCabreraAltimetrik**
please check

### CODE REVIEW

File: force-app/main/default/triggers/TaskTrigger.trigger [hunk 1]
```diff
@@ -0,0 +1,20 @@
+trigger TaskTrigger on Task (before insert,before update,after insert,after update) {
+	
+    if(Trigger.isBefore){
+        if(Trigger.isInsert){
+        	System.debug('isBefore isInsert');
```
Comment (stefaniaoverman) [inline force-app/main/default/triggers/TaskTrigger.trigger:5]: You should make sure that the code you push doesn't include system.debugs nor console.logs

File: force-app/main/default/classes/TaskTriggerHandler.cls [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                for(Task task : c.tasks)
+                {
+                    if(task.IsClosed)
+                    {
+                        if(task.Task_Type__c == 'TO-DO')
+                        {
+                            if(task.ActivityDate != null)
+                                {
+                                   if(task.ActivityDate < Date.today())
+                                   {
```
Comment (stefaniaoverman) [inline force-app/main/default/classes/TaskTriggerHandler.cls]: I'm pretty sure this nested Ifs can be optimized

File: force-app/main/default/classes/TaskTriggerHandler.cls [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            {
+                double contactPoints = c.Points__c == null ? 0 : c.Points__c;
+                
+                for(Task task : c.tasks)
+                {
+                    if(task.IsClosed)
+                    {
+                        if(task.Task_Type__c == 'TO-DO')
+                        {
+                            if(task.ActivityDate != null)
```
Comment (stefaniaoverman) [inline force-app/main/default/classes/TaskTriggerHandler.cls:25]: I'm pretty sure this nested Ifs can be optimized

**Review by stefaniaoverman — CHANGES_REQUESTED** [review]

**Comment by jCabreraAltimetrik** [comment]
Removed System.debug and console.logs; improved upon code removing various nested ifs.

**Pull request closed without merging**